client/src/molecules/Tabs/Tabs.stories.tsx

```diff
@@ -24,7 +24,7 @@ const renderTabs = (darkMode: boolean) => (
         : 'unset'
     }}
   >
-    <TabBar>
+    <TabBar columns='auto auto'>
```

Comment:
Can this be handle internally inside the component rather than be required as a prop when implementing?